### DISCUSSION TITLE
v3.2.0-rc

### DISCUSSION THREAD

**Discussion by REJack** [ColorlibHQ/AdminLTE#3943] [General]
AdminLTE v3.2.0-rc is live, please test it 😄 I've published it on npm with the `next`-tag.

Please report if it works or has some bugs to get it fully released soon.

<hr /><em>This discussion was created from the release <a href='https://github.com/ColorlibHQ/AdminLTE/releases/tag/v3.2.0-rc'>v3.2.0-rc</a>.</em>

**Comment by 5872914** [comment]
终于发布新版了，这个版本时间太长了

**Reply by allisgao** [reply]
同感，很期待。

**Comment by kreic51** [comment]
when using layout-fixed and layout-navbar-fixed, the content moves down the height of the header and the scroll bar appears, is this expected behavior?

It can be seen in the Dashboard v3 page or in one with little content.

with few options in the sidebar you can see it without using layout-fixed.

in version 3.1.0 this does not happen.

**Reply by dfsmania** [reply]
I have also noticed this issue!

**Comment by 5872914** [comment]
how to make url jump out of iframe?

**Comment by spizzo14** [comment]
@REJack check your email please!

**Comment by consigli** [comment]
Bug 1 - if you select "Nav Child Hide on Collapse" for nav sidebar, on mouse over it opens but not shows children.
![hide](https://user-images.githubusercontent.com/1521856/149672398-acc46361-fd3f-4707-aa38-c9529a91e4f7.JPG)

Bug 2 - when you select "Disable Hover/Focus Auto-Expand" on mouser over you can se brand name and menu label appear under the menu
![reopen](https://user-images.githubusercontent.com/1521856/149672532-4257b504-7b33-474d-9344-00c56c1803fc.JPG)

**Comment by Luiggi13** [comment]
If the user select _Disable Hover/Focus Auto-Expand_ and the sidebar is closed, can see the horizontal scroll.
With the option _Disable Hover/Focus Auto-Expand_ enabled maybe the **os-scrollbar-horizontal** must be hidden
![image](https://user-images.githubusercontent.com/7070431/150344711-b8aebc7c-5dee-473a-86a3-019d3db76a4b.png)

**Reply by REJack** [reply]
you did not loaded overlayScrollbars
